rundown/views.py

```diff
@@ -0,0 +1,86 @@
+from rest_framework.views import APIView
+from django.shortcuts import get_object_or_404
+from event.models import Event
+from .models import Rundown
+from .serializers import RundownSerializer
+from rest_framework.response import Response
+import datetime
+
+def validate_rundown_data(rundown_data):
```

Comment:
this is great implementation for validation, but I think you can implement seperation of concern and put it in a diff file, ie: validations.py